templates/intro-services/README.ipynb

```diff
@@ -251,24 +252,24 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Understanding Ray Serve vs Ray cluster config\n",
+    "### Understanding Ray Serve scaling vs Ray cluster configs\n",
     "\n",
-    "When scaling your service, it is important to understand the interaction of the Serve scaling config (i.e., contents of `@serve.deployment`), vs the Ray cluster config (i.e., number of Ray worker nodes). In general, you can think of the Ray cluster config as an upper bound on service scaling, since Ray Serve runs inside the Ray cluster.\n",
+    "When scaling your service, the Serve scaling config, which contains the `@serve.deployment` parameters, interacts with the Ray Cluster config, which contains the number of Ray worker nodes. Generally, the Ray Cluster config is an upper bound on service scaling, because Ray Serve runs inside a Ray Cluster.\n",
     "\n",
-    "For example, suppose the Ray cluster was configured to have at most 100 CPUs, then Serve would only be able to launch up to 100 replicas, no matter the deployment config.\n",
+    "For example, if you configure the Ray Cluster to have at most 100 CPUs, then Serve can only launch up to 100 replicas, regardless of the scaling config.\n",
     "\n",
-    "For this reason, we generally recommend using the \"Auto-select machines\" cluster config for services (this is the default).\n",
+    "For this reason, enable the \"Auto-select machines\" cluster config for services. This setting is on by default.\n",
     "\n",
-    "#### Editing service cluster config\n",
+    "#### Edit a service cluster config\n",
     "\n",
-    "When a service is first created, it will copy the cluster config from the workspace. After that, the service cluster config is decoupled from the workspace and can be edited independently."
+    "When Anyscale first creates a service, it copies the cluster config from the workspace. After that, Anyscale decouples the service cluster config from the workspace and you can edit it independently."
```

Comment:
I'm unclear here whether we mean Ray autoscaling config or a general workspace compute config? Whenever possible, we should default language to compute config unless it's explicitly something else.